.ci/jobs/e2e-tests-snapshot-versions-8x-gke.yml

```diff
@@ -0,0 +1,21 @@
+---
+- job:
+    description: Run ECK E2E tests on GKE every night to verify ECK against Elastic Stack dev 8.x versions
+    name: cloud-on-k8s-e2e-tests-snapshot-versions
```

Comment:
Should we change the name here ?

Also I'm wondering if it makes sense to have only two jobs and avoid to set the version in the job/file name:

* One for stable versions of the stack : `cloud-on-k8s-e2e-tests-stable-versions` , which would test 6.8 to 7.9 in parallel 
* One for dev/snapshot versions: `cloud-on-k8s-e2e-tests-snapshot-versions`, which would test 7.10 and 8 in parallel 

wdyt ?

Reply:
Yes, for the missing name change.

Interesting idea for the two jobs but I'm afraid I won't have time to delve into this now.